sam2/modeling/sam/transformer.py

```diff
@@ -245,9 +240,7 @@ def forward(self, q: Tensor, k: Tensor, v: Tensor) -> Tensor:
 
         dropout_p = self.dropout_p if self.training else 0.0
         # Attention
-
-        with torch.nn.attention.sdpa_kernel(get_sdp_backends(dropout_p)):
```

Comment:
Any particular reason for removing this ? I created the `get_sdp_backends` function to support the newer backends if available on the system being used.